erpnext/public/js/bank_reconciliation_tool/dialog_manager.js

```diff
@@ -444,10 +444,7 @@ erpnext.accounts.bank_reconciliation.DialogManager = class DialogManager {
 				vouchers: vouchers,
 			},
 			callback: (response) => {
-				const alert_string =
-					"Bank Transaction " +
-					this.bank_transaction.name +
-					" Matched";
+				const alert_string = __("Bank Transaction {0} Matched", [this.bank_transaction.name]);
```

Comment:
Improve translation in other languajes, see:

 https://frappeframework.com/docs/v14/user/en/translations#3-blocks